docs/software/programming.md

```diff
@@ -30,69 +32,71 @@ Python is a text based interpreted programming language with object oriented pro
 
 Source: By [www.python.org](http://www.python.org/community/logos/), [GPL](https://commons.wikimedia.org/w/index.php?curid=34991637)
 
-The install option explicitly only installs Python 3.  
-The `pip` resp. `pip3` Python package manager and development headers are included.
+The install option explicitly installs only **Python 3**.  
+The `pip` and `pip3` Python package manager and development headers are included.
 
 To use the `pip` package manager, a typical usage will be `pip3 install -U <module>`.
 
 ***
 
-Wikipedia: <https://wikipedia.org/wiki/Python_(programming_language)>
+- Python website: <https://www.python.org>
+- Python documentation, from beginner to advanced: <https://www.python.org/doc/>
 
 ## Docker
 
-Docker is used to build, ship, and run distributed applications.
-
-<!-- ![Docker logo](../assets/images/dietpi-software-programming-docker1.svg){: width="200px"} -->
```

Comment:
Did you remove the Docker logo intentionally here?

Reply:
by mistake, re-added it

Reply:
Ah, it was commented anyway since we did not get a response from the Docker legal team about it yet. However, it's fine to have the commented element prepared, same for the Docker Compose logo.